k8s/config/prod-oregon.sh

```diff
@@ -25,4 +25,6 @@ export APP_AWS_BUCKET_NAME=developer-portal-prod-178589013767
 export APP_AWS_STORAGE_BUCKET_NAME=devportal-media-prod
 export APP_AWS_BUCKET_REGION=us-west-2
 
+export CELERY_WORKER_REPLICAS=2
```

Comment:
This is probably fine, but we might not need even two workers for ages.

Reply:
Ok works for me, removed the line which should keep it down to just 1

Reply:
Fixed in 987c599893d01b2f5d8a55037867bf3747f6caac